homeassistant/components/hassio/ingress.py

```diff
@@ -116,6 +116,11 @@ async def _handle_request(
         self, request: web.Request, token: str, path: str
     ) -> web.Response | web.StreamResponse:
         """Ingress route for request."""
+        if request.method == "POST":
+            # Allow larger POST requests to add-ons behind ingress
+            request._client_max_size = (  # pylint: disable=protected-access
+                MAX_UPLOAD_SIZE
+            )
         url = self._create_url(token, path)
         data = await request.read()
```

Comment:
Maybe we can stream that like: https://github.com/NabuCasa/hass-nabucasa/blob/master/hass_nabucasa/voice.py#L192